### PR TITLE
Add class for tag `Form` and method `Html::form()`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Tag/Base/NormalTag.php [hunk 1]
```diff
@@ -31,7 +31,12 @@ final protected function renderTag(): string
      */
     final public function open(): string
     {
-        return '<' . $this->getName() . $this->renderAttributes() . '>';
+        return '<' . $this->getName() . $this->renderAttributes() . '>' . $this->prepend();
```
Comment (samdark) [inline src/Tag/Base/NormalTag.php:34]: Looks like append, not prepend.

**Review by samdark — APPROVED** [review]

File: src/Tag/Base/NormalTag.php (hunk 1 above)
Comment (vjik) [reply src/Tag/Base/NormalTag.php:34]: my logic:
- before
- open tag 
- prepend 
- content of tag 
- append
- close tag
- after

Comment (samdark) [reply src/Tag/Base/NormalTag.php:34]: Ah. OK.

**Comment by samdark** [comment]
👍

**Pull request merged**